### ISSUE TITLE
Compilation error for master 0.2.5-dev223

### ISSUE THREAD

**Issue by TheGiamig** [duckdb/duckdb#1389]
Hi, I'm trying to compile the amalgamation version of 0.2.5-dev223.
I compiled duckdb.cpp/hpp with my application code without using the dll. I don't known if this is correct.

The error is: `C1083 Cannot open include file: pcg_uint128.hpp: No such file or directory	duckdb.cpp 42707`.

Maybe I have to set some preprocessor define or something?

Thanks.

**Comment by hannesmuehleisen** [comment]
Ah this is indeed a bug. Thanks for reporting, will fix.

**Comment by TheGiamig** [comment]
I'm compiling on windows 7, vs2015 64bit.

**Comment by Mytherin** [comment]
Should be fixed on master now. Could you verify that it's fixed?

**Comment by TheGiamig** [comment]
Ok, the #include problem is fixed.
Now the compilation stops with the error: `C2872 idx_t: ambiguous symbol`

Version 0.2.5-dev235, but I found the same error also in 0.2.4

**Comment by hannesmuehleisen** [comment]
I think we need a more complete description of what you are doing here. Could you please try to compile a minimal reproducible example of the issue you describe? Thanks!

**Comment by TheGiamig** [comment]
Sure! I'm working on windows 7, vs2015 64bit.
Just right now I downloaded the 0.2.5-dev276 amalgamation version from `https://github.com/cwida/duckdb/releases/download/master-builds/libduckdb-src.zip`
The I created a new project (console application) and added `duckdb.cpp` and `duckdb.hpp` to it.
I defined `_WIN32` and `DUCKDB_BUILD_LIBRARY` in the preprocessor definitions.
The main is simply

> #include "duckdb.hpp"
> 
> using namespace duckdb;
> 
> int main()
> {
> 	DuckDB db(nullptr);
> 	Connection con(db);
> 	auto result = con.Query("SELECT 42");
> 	result->Print();
> 
>    return 0;
> }

The compiler error is `C2872 idx_t: ambiguous symbol`.
I can found two definitions for idx_t: duckdb.hpp line 91 and line 7524.
Cheers
